plugins/modules/keycloak_realm_key.py

```diff
@@ -371,7 +371,7 @@ def main():
     parent_id = module.params.get('parent_id')
 
     # Get a list of all Keycloak components that are of keyprovider type.
-    realm_keys = kc.get_components(urlencode(dict(type=provider_type, parent=parent_id)), parent_id)
+    realm_keys = kc.get_components(urlencode(dict(type=provider_type)), parent_id)
```

Comment:
Hmm, I wonder whether this is something that changed over time, and thus is still needed for older Keycloak versions.

Reply:
a) looking at the oldest available docs (keycloak 21.0), this has always been optional
b) it is filtered to this realm scope anyways since .get_components requests from GET /{realm}/components